File: 268-MissingNumber/268-MissingNumber.js
// Last updated: 10/03/2026, 10:03:23
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let realsum=0;
    let n=nums.length;
      let exsum = n * (n + 1) / 2;

    for(let i=0;i<nums.length;i++){
       realsum+=nums[i];
    }
    return exsum-realsum;
};

File: 268-MissingNumber/268-MissingNumber.cpp
// Last updated: 03/08/2026, 10:12:19
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        int expected=n * (n+1)/2;

        int actual =0;
        for(int num:nums){
            actual+=num;
        }
    return expected -actual;
    }
};